### PR TITLE
enhance/fix invalid argument exception string payload

### PR DESCRIPTION
**Pull request by domsj**

### CODE REVIEW

File: src/unix/lwt_bytes.ml [hunk 1]
```diff
@@ -60,23 +60,23 @@ let blit_from_bytes src_buf src_ofs dst_buf dst_ofs len =
   if (len < 0
       || src_ofs < 0 || src_ofs > Bytes.length src_buf - len
       || dst_ofs < 0 || dst_ofs > length dst_buf - len) then
-    invalid_arg "String.blit"
+    invalid_arg "Lwt_bytes.blit"
```
Comment (aantron) [inline src/unix/lwt_bytes.ml]: Should this be `"Lwt_bytes.blit_from_bytes"`?

File: src/unix/lwt_bytes.ml [hunk 2]
```diff
@@ -60,23 +60,23 @@ let blit_from_bytes src_buf src_ofs dst_buf dst_ofs len =
...
   else
     unsafe_blit_from_bytes src_buf src_ofs dst_buf dst_ofs len
 
 let blit_to_bytes src_buf src_ofs dst_buf dst_ofs len =
   if (len < 0
       || src_ofs < 0 || src_ofs > length src_buf - len
       || dst_ofs < 0 || dst_ofs > Bytes.length dst_buf - len) then
-    invalid_arg "String.blit"
+    invalid_arg "Lwt_bytes.blit"
```
Comment (aantron) [inline src/unix/lwt_bytes.ml]: ...and this `"Lwt_bytes.blit_to_bytes"`?

File: src/unix/lwt_bytes.ml (hunk 1 above)
Comment (domsj) [inline src/unix/lwt_bytes.ml]: yes

File: src/unix/lwt_bytes.ml (hunk 2 above)
Comment (domsj) [inline src/unix/lwt_bytes.ml]: and yes
see new commit

**Comment by aantron** [comment]
Thanks again!

**Pull request merged**